psycopg_pool/psycopg_pool/base.py

```diff
@@ -19,6 +19,24 @@
     from psycopg._connection_base import BaseConnection
 
 
+class _PoolStats(TypedDict):
```

Comment:
Mainly making this private since I'm not sure it makes sense to live here from an interface POV and I'm not seeing an existing convention for `.types` module that contain type aliases (only have seen `psycopg.abc`), but let me know if you think there's a good place this could live and not be private.